offline/federation_preparation.md

```diff
@@ -0,0 +1,69 @@
+## RabbitMQ
+
+There are two ways to deploy the RabbitMQ cluster:
+
+Method 1
+---
+## Install RabbitMQ inside kubernetes cluster with the help of helm chart
+
+To install the rabbitmq,
+First copy the value and secret file:
+```
+cp ./values/rabbitmq/prod-values.example.yaml ./values/rabbitmq/values.yaml
+cp ./values/rabbitmq/prod-secrets.example.yaml ./values/rabbitmq/secrets.yaml
+```
+By default this will use ephemeral storage, to use the local persistance storage in kubernetes nodes, please refer to this doc - offline/local_persistance_storage.md
+
+Now, update the `./values/rabbitmq/values.yaml` and `./values/rabbitmq/secrets.yaml` with correct values as per needed.
+
+Deploy the rabbitmq helm chart -
+```
+d helm upgrade --install rabbitmq ./charts/rabbitmq --values ./values/rabbitmq/values.yaml --values ./values/rabbitmq/secrets.yaml
+```
+
+Method 2
+---
+
+## Installing RabbitMQ outside kubernets cluster with ansible playbook
+
+Add the nodes in which you want to run rabbitmq to the `[rmq-cluster]` group in the `ansible/inventory/offline/hosts.ini` file. Also, update the `ansible/roles/rabbimq-cluster/defaults/main.yml` file with the correct configurations for your environment.
+
+You should have following entries in the /ansible/inventory/offline/hosts.ini file for example:
```

Comment:
An explanation what these *entries* mean would be great. `rabbitmq_network_interface` should be pretty clear. But, how do I figure out `ansnode1` et. al. for my setup?